src/content/docs/getting-started/first-program.mdx

```diff
@@ -4,20 +4,20 @@ title: Creating your first program
 
 import { Steps } from "@astrojs/starlight/components";
 
-Creating a new Membrane program is easy.
+In this "Hello World" video we create a program and invoke its `run` [action](/concepts/schema#actions). The results of
+the invocation can be seen in the logs.
+
+<video src="/cloud-assets/new-program.mp4" muted autoplay loop></video>
+
+Creating a new Membrane program from <a href="https://membrane.io/ide" target="_blank">the IDE</a> takes two clicks:
 
 <Steps>
 
-1. Open the <a href="https://membrane.io/ide" target="_blank">Membrane IDE</a>
-2. Click the `NEW` button and select which type of program you'd like to create
-3. Invoke the `run` action in your new program
+1. Click `NEW PROGRAM`
+2. Click `Action` to create a simple program with just one action. (change the name if you want)
```

Comment:
I'm planning to change the templates soon per Andrew's feedback—I'll edit the docs accordingly when I do